pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -150,6 +150,35 @@ type Placement struct {
 	// +optional
 	ClusterAffinity *ClusterAffinity `json:"clusterAffinity,omitempty"`
 
+	// OrderedClusterAffinity represents multiple cluster groups.
```

Comment:
I would prefer use "set of clusters" instead of "cluster groups", to align with Placement.ClusterAffinity field description.